README.rst

```diff
@@ -114,7 +111,6 @@ EinstienPy is usually tested on Linux, Windows and OS X on Python
 Platform        Site          Status
 ==============  ============  ===================
 Linux           CircleCI      |circleci|
-OS X            Travis CI     |travisci|
```

Comment:
Add the respective line here?

Reply:
https://github.community/t5/GitHub-Actions/Missing-documentation-on-GitHub-Actions-badges/m-p/30970#M628